LinkedListClass/LinkedList.cs

```diff
@@ -0,0 +1,92 @@
+﻿namespace LinkedListClass
+{
+    public class LinkedList
```

Comment:
You don't to create a new custom class, use the LinkedList<> from System.Collections

Reply:
This one blew up my brain 🤯 Hopefully, fixed.